src/SFA.DAS.EmployerAccounts.Web/Startup.cs

```diff
@@ -49,57 +50,50 @@ public void Configuration(IAppBuilder app)
 
             app.UseCookieAuthentication(new CookieAuthenticationOptions
             {
-                AuthenticationType = "Cookies",
+                AuthenticationType = CookieAuthenticationDefaults.AuthenticationType,
                 ExpireTimeSpan = new TimeSpan(0, 10, 0),
                 SlidingExpiration = true
             });
 
             app.UseCookieAuthentication(new CookieAuthenticationOptions
             {
-                AuthenticationType = "TempState",
+                AuthenticationType = TempState,
                 AuthenticationMode = AuthenticationMode.Passive
             });
 
-            app.UseCookieAuthentication(new CookieAuthenticationOptions
-            {
-                AuthenticationType = "Staff",
-            });
-
-            app.UseCookieAuthentication(new CookieAuthenticationOptions
-            {
-                AuthenticationType = "Employer",
-                ExpireTimeSpan = new TimeSpan(0, 10, 0),
-                SlidingExpiration = true
+            app.UseSupportConsoleAuthentication(new SupportConsoleAuthenticationOptions 
+            { 
+                AdfsOptions = new ADFSOptions
+                {
+                   
+                    MetadataAddress = config.AdfsMetadata , 
+                    Wreply = config.EmployerAccountsBaseUrl , 
+                    Wtrealm = config.EmployerAccountsBaseUrl ,
+                    BaseUrl = config.Identity.BaseAddress,
+                    ClientId = config.Identity.ClientId,
```

Comment:
ClientId, ClientSecret  etc... are not required for the supportConsoleAuthentication